app/views/user/dashboard/show.html.erb

```diff
@@ -1,12 +1,14 @@
 <h1>Welcome <%= current_user.name %>!</h1>
 
-<p>Delete my account:</p>
+Edit my account:
+<%= button_to 'Edit', edit_user_path(current_user) %>
 
+<p>Delete my account:</p>
```

Comment:
The User Dashboard should now be the only place a user can edit or delete their account